### ISSUE TITLE
NTR: Intrinsic factor or antiparietal antibodies present (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8704]
New term request Intrinsic factor or antiparietal antibodies present (segal_050616183501):   UMLS: 0853670

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000343
- label: Antiparietal cell antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:34913949
- def: The presence of autoantibodies (immunoglobulins) in the serum that react against parietal cell antigens.
- comment: This finding may be observed in autoimmune gastritis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8704
- synonyms:
